scripts/components/openblas.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 # OpenBLAS (SuiteSparse dependency)
 CUR_DIR=$WORK_DIR/OpenBLAS
+: "${OPENBLAS_CMAKE_FLAGS:=-DCMAKE_C_FLAGS=-DFORCE_HASWELL}"
```

Comment:
This may or may not need to be changed on Mac (just set OPENBLAS_CMAKE_FLAGS to something else like in the Android build)

Reply:
It seemed to work fine for me as is